File: C++/BubbleSort.cpp
#include <bits/stdc++.h>
using namespace std;

void swap(int *x, int *y)
{
    int temp = *x;
    *x = *y;
    *y = temp;
}
void bubbleSort(int arr[], int n)
{
    for (int i = 0; i < n - 1; i++)
    {
        for (int j = 0; j < n - i - 1; j++)
        {
            if (arr[j] > arr[j + 1]){
                swap(&arr[j], &arr[j + 1]);
            }
        }
    }
}

int main() 
{ 
    int arr[] = {5,9,8,4,3,2,1,10,7}; 
    int n = sizeof(arr)/sizeof(arr[0]);
    cout << "Before Sort: ";
    for(int i = 0; i < n; i++){
        cout << arr[i] << " " ;
    }
    bubbleSort(arr, n); 
    cout << "After Sort: ";
    for(int i = 0; i < n; i++){
        cout << arr[i] << " ";
    }
    return 0; 
}

File: Python/BubbleSort.py
def bubbleSort(arr,n):
    for i in range(n):
        for j in range(0, n-i-1):
            if arr[j] > arr[j+1] :
                arr[j], arr[j+1] = arr[j+1], arr[j]
arr = [5,4,8,2,1,7,6,10,3]
print("Before Sort: ")
for x in range(len(arr)):
    print(arr[x])
bubbleSort(arr,len(arr))
print("After Sort: ")
for x in range(len(arr)):
    print(arr[x])